### DISCUSSION TITLE
Errors When Importing DLL's

### DISCUSSION THREAD

**Discussion by JohnessyAI** [srcnalt/RPM-Smart-NPC#4] [General]
Hey mate, 

I was wondering if you could help with these errors im getting. I know it has to have something to do with an existing library, or a wrong version etc. 

Here is the first one - 

`Error: Could not load signature of Amazon.Polly.Model.IListSpeechSynthesisTasksPaginator:get_Responses due to: Could not resolve type with token 010000a4 (from typeref, class/assembly System.Collections.Generic.IAsyncEnumerable`1, Microsoft.Bcl.AsyncInterfaces, Version=1.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51) assembly:Microsoft.Bcl.AsyncInterfaces, Version=1.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51 type:System.Collections.Generic.IAsyncEnumerable`1 member:(null) signature:<none>
`

And Second

Error: Could not load signature of Amazon.Runtime.IPaginator`1[TResponse]:PaginateAsync due to: Could not resolve type with token 010000a4 (from typeref, class/assembly System.Collections.Generic.IAsyncEnumerable`1, Microsoft.Bcl.AsyncInterfaces, Version=1.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51) assembly:Microsoft.Bcl.AsyncInterfaces, Version=1.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51 type:System.Collections.Generic.IAsyncEnumerable`1 member:(null) signature:<none>

Then i get these - Unloading broken assembly Assets/Plugins/Polly/AWSSDK.Polly.dll, this assembly can cause crashes in the runtime

This has me totally stumped! 

Also hope the sponsor showed a little bit of gratitude for what you have built bud!

**Comment by JohnessyAI** [comment]
Now these are imported the exact way you did on the video, Has my brain baffled.

**Comment by srcnalt** [comment]
Hi @JohnessyAI, thank you so much for sponsoring. I am more than happy to help regardless.
I was going to ask about the versions of the packages you imported but you already commented that you followed the exact tutorial.
Can you tell me which version of unity an which scripting backend are you using? In the example I used `Unity 2021.3.5f1` and `Mono` scripting backend and `.NET Standard 2.1` API compatibility level.

**Reply by JohnessyAI** [reply]
Hey man, Thanks for helping bud! appreciate it loads. The sponsoring wasnt for help was literally just because of what you have built and made open! 

The version is 2020.3.21f1 
Scripting Backend .net 2 we also tried 4x but same issue.

**Comment by JohnessyAI** [comment]
Wait i didnt realise i was so out of date! Updating

**Comment by JohnessyAI** [comment]
Mate you are a legend! It works and im an idiot! I updated this thing loads of times, had no clue i was meant to change settings and what version in the hub!

**Reply by srcnalt** [reply]
Glad to hear that it works now!
